Casks/camo-studio.rb

```diff
@@ -1,15 +1,32 @@
 cask "camo-studio" do
-  version "1.2.3,139"
-  sha256 "dc5983b84f2a0b5500fa44e59acf744228cf03d3e10497b05ba09944d6acfa08"
+  version "1.2.4,144"
+  sha256 "37aea984109f4c8bf6f007a8bd8cb55de2e745773684c8cca3d736b421023ce0"
 
   url "https://reincubate.com/res/labs/camo/Camo%20Studio%20#{version.before_comma}%20%28#{version.after_comma}%29%20%5BRelease%5D.app.zip"
-  appcast "https://uds.reincubate.com/release-notes/camo/?format=sparkle"
   name "Camo Studio"
+  desc "Use your phone as a pro webcam to look amazing on video calls"
```

Comment:
```suggestion
  desc "Use your phone as a webcam"
```

The extra text is what we describe as “marketing fluff” (no offence!). Descriptions should be brief in telling *objectively* what the app does or is for.

Reply:
That makes sense. Was referencing our public copy. 

The last fragment may indeed be extraneous fluff, but I would consider 'pro' as useful; Camo Studio offers many controls that go beyond what the proposed would describe. Perhaps there's a better, more precise description for this field without the fluff.

Reply:
What makes it a “pro” webcam? What’s the most important feature of those?

Reply:
I've just updated the description with a short list of some of our most important features, to avoid any potential subjective adjectives.